src/main/java/com/fismobile/appium/base/BaseClass.java

```diff
@@ -32,23 +32,21 @@
 public class BaseClass {
 
     private static WebDriver driver;
-    private TestContext testContext;
     private Logger logger = LogManager.getLogger(BaseClass.class);
 
-
-    private WebDriver setup()
+    @Parameters({"platform","runOn"})
+    @BeforeClass
+    public void setup(@Optional("web") String platform, @Optional("chrome") String runOn)
     {
```

Comment:
Curly brackets should go in the same line of the method name and with a single space separation. This comment applies for all the curly brackets (if, for, etc).

If the method name with its arguments or the sentence is too long, you can add line breaks for a better reading.